### DISCUSSION TITLE
Is my score a bug? - 16/7/2024

### DISCUSSION THREAD

**Discussion by bartgoselink** [cssbattle/forum#393] [Bugs reports]
It seems like I recreated the target 100% match but it states 69%. Is there anything I should change? I already resubmitted, got the same results. 

`<style>*{background:#4F77FF;*{background:#fff;width:200px;height:200px;border-radius:40px;margin:-100px 0 0 -100px;box-shadow:400px 0 0 white,400px 300px 0 white,0 300px 0 #fff;`

<img width="1727" alt="Screenshot 2024-07-16 at 10 10 43" src="https://github.com/user-attachments/assets/13358acf-ca72-4c95-9d48-8a5b2b8c923a">

**Comment by chinchang** [comment]
You used the wrong white color mate :)

**Reply by bartgoselink** [reply]
🤦‍♂️ how could I have missed that, thanks @chinchang
